oc-aliases.sh

```diff
@@ -80,29 +87,29 @@ function load_connection() {
 		echo "load_connection <connection.json>"
 	else
 		gettoken
-		curl -k -H "Authorization: Bearer ${TOKEN}" https://${SD_HOST:?environment variable must be set}/connmgr/v1/connections -X POST -H "Content-type: application/json" -d@"$1"
+		curl -k -H "Authorization: Bearer ${TOKEN}" https://${SD_CONSOLE_ROUTE:?environment variable must be set}/connmgr/v1/connections -X POST -H "Content-type: application/json" -d@"$1"
 	fi
 }
 
 # Get all running pods
-alias allpods='oc get pods --all-namespaces'
-alias kp='oc get pods --all-namespaces'
+alias allpods='oc -n ${PROJECT_NAME} get pods'
+alias kp='oc -n ${PROJECT_NAME} get pods'
 
 # Fetch a valid authentication token
-alias gettoken="export TOKEN=\$(curl -k -u \${SD_USER:?environment variable must be set}:\${SD_PASSWORD:?environment variable must be set} https://\${SD_HOST:?environment variable must be set}/auth/v1/token -I | grep X-Auth-Token |cut -f 2 -d \" \")"
+alias gettoken='export TOKEN=$(curl -s -k -u ${SD_USER:-sdadmin}:${SD_PASSWORD:?environment variable must be set} https://${SD_CONSOLE_ROUTE:?environment variable must be set}/auth/v1/token -I | grep -i X-Auth-Token | cut -f 2 -d " ") | xargs'
 
 # Curl with authorization token
-alias tcurl="curl -k -H \"Authorization: Bearer \${TOKEN}\""
-alias tcurl_json="curl -k -H \"Authorization: Bearer \${TOKEN}\" -H \"Content-type: application/json\""
+alias tcurl='curl -s -k -H "Authorization: Bearer ${TOKEN}"'
+alias tcurl_json='curl -k -H "Authorization: Bearer ${TOKEN}" -H "Content-Type: application/json"'
 
 #COS-SCANNER aliases for running scanner scripts inside docker
-alias cos-notify='oc exec -it --namespace=spectrum-discover $(oc get pods -n=spectrum-discover -l app=spectrum-discover-connmgr -o=jsonpath='{.items[0].metadata.name}') -- bash -c "python3 -m scanners.cloud_scanner.main_notifier"'
-alias cos-replay='oc exec -it --namespace=spectrum-discover $(oc get pods -n=spectrum-discover -l app=spectrum-discover-connmgr -o=jsonpath='{.items[0].metadata.name}') -- bash -c "python3 -m scanners.cloud_scanner.main_replay"'
+alias cos-notify='oc -n ${PROJECT_NAME} exec -c spectrum-discover -it $(oc -n ${PROJECT_NAME} get pods -l role=connmgr -o name) -- bash -c "python3 -m scanners.cloud_scanner.main_notifier"'
+alias cos-replay='oc -n ${PROJECT_NAME} exec -c spectrum-discover -it $(oc -n ${PROJECT_NAME} get pods -l role=connmgr -o name) -- bash -c "python3 -m scanners.cloud_scanner.main_replay"'
```

Comment:
Adding "-l app=isd-connmgr" could be a good idea

Reply:
I think that is an unnecessary additional overhead